### PR TITLE
Revert "[Doc] Mark multicluster allocation feature as stable"

### PR DESCRIPTION
**Pull request by pooneh-m**
Reverts googleforgames/agones#1831

### CODE REVIEW

**Review by aLekSer — APPROVED** [review]
Agree with Robert we need to add website shortcodes `releaseVersion` and expiry.

File: site/content/en/docs/Guides/feature-stages.md [hunk 1]
```diff
@@ -26,12 +26,20 @@ The current set of `alpha` and `beta` feature gates are:
 
 | Feature Name | Gate    | Default | Stage | Since |
 |--------------|---------|---------|-------|-------|
+{{% feature expiryVersion="1.10.0" %}}
```
Comment (aLekSer) [inline site/content/en/docs/Guides/feature-stages.md:29]: This would break the table. Let's do a separate Revert + New version. WDYT?

**Review by aLekSer — CHANGES_REQUESTED** [review]

File: site/content/en/docs/Guides/feature-stages.md (hunk 1 above)
Comment (pooneh-m) [reply site/content/en/docs/Guides/feature-stages.md]: Done

**Review by roberthbailey — APPROVED** [review]

**Review by aLekSer — APPROVED** [review]

**Comment by aLekSer** [comment]
429 to Github error

**Pull request merged**